src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -339,6 +339,7 @@ protected function describeContainerDefinition(Definition $definition, array $op
      */
     protected function describeContainerAlias(Alias $alias, array $options = array())
     {
+        $options['output']->setDecorated(false);
```

Comment:
This test is part of the FrameworkBundle TextDescriptor that is not responsible of the console output style, so IMO we should not test the output style of a comment here, I written a test for that on the corresponding class (SymfonyStyle). Plus, this test needs to be adapted each time we made a change on the `comment()` method, that is first confusing.